overlays/build-tools-in-build-inputs.nix

```diff
@@ -9,10 +9,109 @@ let
   inherit (import ../lib { inherit lib; }) checkMkDerivationFor;
 
   buildTools = [
+    "antlr"
+    "appstream-glib"
+    "asciidoc"
+    "asciidoctor"
+    "autoconf"
+    "autogen"
+    "automake"
+    "automoc4"
+    "autoreconfHook"
+    "autoreconfHook"
+    "bazel"
+    "bc"
+    "bdf2psf"
+    "bison"
+    "bmake"
+    "breakpointHook"
+    "bzip2"
+    "bundler"
+    # "darwin.cctools"
```

Comment:
I guess we could do `lib.attrByPath (lib.splitString "." tool) (throw "‘${tool}’ does not exist in Nixpkgs.") prev`.

Reply:
This mainly does not work because it is nested.

Reply:
Yeah, replacing the `prev.${tool}` in the `checkDerivation` definition below with the code above should fix that.

Reply:
Added that.